config/spring.rb

```diff
@@ -4,3 +4,11 @@
   tmp/restart.txt
   tmp/caching-dev.txt
 ].each { |path| Spring.watch(path) }
+Spring.after_fork do
```

Comment:
oops - I don't think this debugger stuff is intended for here

Reply:
Oops, it ended up getting  pushed.